docs/getting_started/use-cases.rst

```diff
@@ -196,12 +196,12 @@ Adding Options
 --------------
 
 By default, some of the commands will come with singularity options. For example,
-a container intended for gpu is always going to give you the ``--nv`` flag. However,
+a container intended for gpu should have the ``--nv`` flag. However,
```

Comment:
@surak I don’t think tensorflow is one of the containers flagged for gpu?

Reply:
I think probably in your settings.yaml you didn’t turn on gpu so it wasn’t installed with it - can we check that?

Reply:
No, but PyTorch is -it's an nvidia one and it was compiled with gpu support and yet, it's not flagged as such. In any case, if one brings a container from elsewhere, one won't know that the module should be flagged, right?

Reply:
If PyTorch needs the features gpu flag we can add it. But the center or user still needs to set gpu to true in the settings.yaml.

Reply:
See here: https://github.com/singularityhub/singularity-hpc/blob/bfb2c33796d60e291639e0725f00b3bcff084aa7/shpc/settings.yml#L63

Reply:
Hmm interesting - so the hope was that there would be two controllers - one for the center / user to be able to say "always add these flags given that the container says they are supported (the settings here) and then the "on the fly" add this flag given that the center hasn't done that. I do think the current change of the PR is much better and I'd like to merge!